src/pudl_catalog/__init__.py

```diff
@@ -13,10 +13,11 @@
 
 INTAKE_BUCKET = "intake.catalyst.coop"
 # Ideally we would set this version automatically using setuptools_scm...
-CATALOG_VERSION = "v0.1.1"
+CATALOG_VERSION = "v2022.11.30"
```

Comment:
I've been imagining that we would have the `dev` branch of the `pudl-catalog` always point at the last successful nightly build outputs (which show up at `/dev`), and then do tagged releases of `pudl-catalog` that mirror the main pudl / data releases (e.g. `v2022.11.30`). So if someone wants the development / beta / work-in-progress data, they can always install from `dev` and if they want a stable version, they would install from pip/conda.  Do you see any reason why that wouldn't work?

Ideally I'd like to set up this repo to bump the version to track PUDL, and do a release whenever the PUDL repo has a new tagged release + successful nightly build.

Reply:
So maybe it makes sense to set up this pattern now? Like have the URL here point at `dev` and merge these changes into `main` (or a branch off of `main`) and then create a "release" on main with the new stable version / URL.

Reply:
I think that makes sense. You're saying on `main`, `CATALOG_VERSION = "dev"` and we'll have tagged branches where 

```python
CATALOG_VERSION = "v2022.11.30" # or any future tags
```

So if folks want access to dev data through the catalog, they'll install from the main or dev branch?

Reply:
I guess it would work to have the `main` branch produce a catalog with `dev` as the version, and then just for released versions have it be like `v2022.11.30`? But then whenever there was a released / tagged commit that happened to be `main` you wouldn't get `dev` you'd get whatever the released version was.

Reply:
Got it so for this PR I should set `CATALOG_VERSION = dev`?

Reply:
If I understand correctly, I think we should always have `CATALOG_VERSION = dev` unless it's a tagged release, in which case the release version and catalog version should be the same and match the PUDL data release version that they refer to.